src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -195,54 +195,84 @@ export default function FacilityOrganizationIndex({
               {isTopLevel && children.length > 0 && (
                 <div>
                   <TooltipProvider>
+                    {isMobile ? (
```

Comment:
Let's bring TooltipProvider inside, doesn't make sense to wrap the button on regular screen with TooltipProvider.